app/main/views/services.py

```diff
@@ -214,7 +214,7 @@ def create_new_draft_service(framework_slug):
     lot = request.form.get('lot', None)
 
     if not lot:
-        question = content_loader.get_builder(framework_slug, 'edit_submission').get_question('lot')
+        question = content_loader.get_question(framework_slug, 'services', 'lot')
```

Comment:
This breaks with the current API because it still has lots in the schema - trying to create a new draft gives, e.g.  `{'lot': "'paas' is not one of ['PaaS']"}`